selfdrive/car/honda/values.py

```diff
@@ -35,7 +35,6 @@ class CruiseButtons:
 
 class CAR:
   ACCORD = "HONDA ACCORD 2T 2018"
```

Comment:
Do we want to remove the 2T in the name to generalize it?

Reply:
yes